charts/snapshot-controller/templates/deployment_validation_webhook.yaml

```diff
@@ -82,4 +82,14 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- if kindIs "bool" .Values.webhook.hostNetwork }}
+      hostNetwork: {{ .Values.webhook.hostNetwork }}
+      {{- end }}
+      {{- with .Values.webhook.dnsConfig }}
+      dnsConfig:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.webhook.dnsPolicy }}
+      dnsPolicy: {{ . }}
+      {{- end }}
```

Comment:
Same suggestion as above

Reply:
Correction, `CSI Snapshot Controller` and `Amazon EBS CSI Driver` are two separate add-ons.